cds/modules/webhooks/tasks.py

```diff
@@ -431,6 +424,54 @@ def progress_updater(current_frame):
         db.session.commit()
 
     @classmethod
+    def _time_position(cls, duration, frames_start=5, frames_end=95,
+                       frames_gap=10):
+        """Calculate time positions."""
+        duration = float(duration)
+        time_step = duration * frames_gap / 100
+        start_time = duration * frames_start / 100
+        end_time = (duration * frames_end / 100) + 0.01
```

Comment:
Probably not relevant, but it is not clear to why + 0.01

Reply:
we should ask to @omelkonian :smile:

Reply:
Hmm, I think the reason is that you want to include the frame that may be exactly at `end_time`, so you go a tiny bit further than that, in order to include it.

Anyway, ffmpeg is somewhat inconsistent between versions, so you should play around with it and try to determine its behaviour in the corner cases.